trin-core/src/types/header.rs

```diff
@@ -28,44 +29,76 @@ pub struct Header {
     /// Block receipts root.
     pub receipts_root: H256,
     /// Block bloom filter.
-    pub log_bloom: Bloom,
+    pub logs_bloom: Bloom,
     /// Block difficulty.
     pub difficulty: U256,
     /// Block number.
+    #[serde(deserialize_with = "de_hex_to_u64")]
     pub number: u64,
     /// Block gas limit.
     pub gas_limit: U256,
     /// Block gas used.
     pub gas_used: U256,
     /// Block timestamp.
+    #[serde(deserialize_with = "de_hex_to_u64")]
```

Comment:
Same as above, switch to `ethereum_types::U64`.

Reply:
Same as above, I think it's simpler to convert to `u64` here, rather than converting whenever this field is used in the codebase